testsuite/docs/cucumber-steps.md

```diff
@@ -613,6 +625,44 @@ For example:
   When I call actionchain.add_package_install()
 ```
 
+<a name="b13" />
+
+#### Virtualization
+
+* Create a test virtual machine on a given host
+
+The virtual machine is created without SUSE Manager, directly on the virtual host
+using `qemu-img` and `virt-install`
```

Comment:
Please move the 2 lines above to a different file (probably `optional.md`), or drop them. The subject here is the cucumber steps, not how to set up the test suite.

Reply:
Hum, no... this has nothing to do with setup: it's a step comment